tf2onnx/tfonnx.py

```diff
@@ -1369,7 +1369,7 @@ def tensorflow_onnx_mapping(g, continue_on_error, custom_op_handlers):
 def tf_optimize(sess, inputs, outputs, graph_def):
     """Optimize tensorflow graph for inference."""
     transforms = [
-        # "fold_constants(ignore_errors=true)",
```

Comment:
fold_constants on some graphs runs into a tensorflow issue ... that is the reason why it is off.

Reply:
maybe I can enable it once use decides to enable the optimization? because current optimization depends on this, otherwise, lots of transpose remains. but eventually I will have a try when it is disabled.

Reply:
I think that would be ok. 
The error message you'd get is 'expects to be colocated with unknown node ' and there are some reports of this (I thought I had seen some detailed tensorflow issue on this but can't find it anymore).
We do have models that runs into this but I have not looked closer ... maybe time to understand what the root cause is.